components/Article/Codebox/inline.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import InlineCode from './InlineCode';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof InlineCode> = {
+  title: 'InlineCode',
+  component: InlineCode,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof InlineCode>;
+
+const code = 'const a = 1;';
+
+export const Default: Story = {
+  args: {
+    children: <code>{code}</code>,
```

Comment:
Also do the same with here (with the decorator) so we can actually customise the code xD